src/ResourceManager/RecoveryServices/RecoveryServices.Backup.sln

```diff
@@ -25,6 +25,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Commands.Common.Authenticat
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Commands.RecoveryServices.Backup.Cmdlets", "Commands.RecoveryServices.Backup\Commands.RecoveryServices.Backup.Cmdlets.csproj", "{3B3E879A-F856-46BF-AFF9-8AD6760CF7B9}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Commands.Profile", "..\Profile\Commands.Profile\Commands.Profile.csproj", "{142D7B0B-388A-4CEB-A228-7F6D423C5C2E}"
```

Comment:
Please revert the changes to this file - It seems like all project you added are unnecessary (you don't use profile cmdlets so you don't need profile, and all of the Commands.Common.<RP> module are unnecessary unless you use them specifically).